src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -1,336 +1,503 @@
--- NOTE: This script DROPS AND RECREATES all database objects.
 -- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
 
 /*************************************************************
-    Configure database
```

Comment:
Why was this section removed?

Reply:
This configuration is already part of [baseSchema.sql](https://github.com/microsoft/healthcare-shared-components/blob/master/src/Microsoft.Health.SqlServer/Features/Schema/Migrations/BaseSchema.sql#L5) which is executed before executing the initialization schema script so its redundant over here.